freshmaker/config.py

```diff
@@ -401,6 +399,16 @@ class Config(object):
             'type': str,
             'default': 'all',
             'desc': 'vcr mode for recording lightblue queries'
+        },
+        'pyxis_server_url': {
+            'type': str,
+            'default': '',
+            'desc': 'Server URL of Pyxis.'
+        },
+        'pyxis_index_image_organization': {
+            'type': str,
+            'default': 'redhat-operators',
```

Comment:
This is internal only and since this is an upstream repo, I would rather this be set to an empty string and can be configured on the actual Freshmaker deployment.